### DISCUSSION TITLE
Expected behaviour of isMatch

### DISCUSSION THREAD

**Discussion by geoffreydhuyvetters** [js-fns/date-fns#2744] [Q&A] [answered]
We're trying to use isMatch in a codebase.

Isn't expected behaviour here to return false instead of true?

`isMatch('0:20', "HH:mm");` 

since HH requires double digits?

**Comment by tan75** [comment, marked as answer]
hi  @geoffreydhuyvetters 
currently, the `isMatch` function is not strict. The strict version of it is coming with v3.

**Reply by geoffreydhuyvetters** [reply]
Clear, thanks for the answer!
